### PR TITLE
ip addresses: Add the Healthcheck parameters

### PR DESCRIPTION
**Pull request by mcorbin**
- Add the Healthcheck field in IPAddress
- Add the healthcheck parameters in the AssociateIPAddress call

Story: [EIP health checking in egoscale](https://app.clubhouse.io/exoscale/story/1953)

### CODE REVIEW

File: addresses.go [hunk 1]
```diff
@@ -77,10 +89,15 @@ func (ipaddress IPAddress) Delete(ctx context.Context, client *Client) error {
 
 // AssociateIPAddress (Async) represents the IP creation
 type AssociateIPAddress struct {
-	IsPortable *bool `json:"isportable,omitempty" doc:"should be set to true if public IP is required to be transferable across zones, if not specified defaults to false"`
-	NetworkdID *UUID `json:"networkid,omitempty" doc:"The network this ip address should be associated to."`
-	ZoneID     *UUID `json:"zoneid,omitempty" doc:"the ID of the availability zone you want to acquire an public IP address from"`
-	_          bool  `name:"associateIpAddress" description:"Acquires and associates a public IP to an account."`
+	Interval    int64  `json:"interval,omitempty" doc:"healthcheck definition: time in seconds to wait for each check. Default: 10, minimum: 5"`
```
Comment (falzm) [inline addresses.go]: I'd prefix the healthcheck-related struct fields with `HealthcheckXXX` for a more explicit properties separation (even if the actual API parameters are not).

Comment (mcorbin) [reply addresses.go]: Done ;)

Comment (falzm) [reply addresses.go]: Perfect! 👌

**Comment by falzm** [comment]
> LGTM but @mcorbin can you add some test please thanks

I'm not sure this is necessary in this case: the CLI has virtually no code involved other than wrapping the API call, for which there are already integration tests running periodically...

**Pull request merged**